torch_glow/tests/nodes/quantized_linear_test.py

```diff
@@ -13,6 +13,8 @@ def test_quantized_linear_packed():
     dq = torch.nn.quantized.DeQuantize()
     linear = torch.nn.Linear(5, 5)
 
+    # all weights are the same
+    # therefore is not rowwise quantized
```

Comment:
What's the significance of this?

Reply:
To tell the different between this test and the new added test, in case we want to modify the test in the future.